climada/hazard/tc_tracks_synth.py

```diff
@@ -39,9 +39,9 @@ def calc_perturbed_trajectories(tracks,
                                 nb_synth_tracks=9,
                                 max_shift_ini=0.75,
                                 max_dspeed_rel=0.3,
-                                max_ddirection=np.pi / 180,
+                                max_ddirection=np.pi / 360,
                                 autocorr_dspeed=0.85,
-                                autocorr_ddirection=0.85,
+                                autocorr_ddirection=0.5,
```

Comment:
I do not see a problem with changing the default values in principles. I am not really convinced that the new defaults are much better (they do not hold up to a real scientific inquiry) though. What happens to the other basins? I did a similar analysis for all the basins, and some small changes in the values were needed for all of them. Hence, I suggest that if, as you say "I think this improvement should be included in the develop branch asap and possibly in the upcoming release, to avoid users playing around with an unrealistic set of parameter values.", then it is best to do this for all basins accordingly. It does not make much sense to argue for very fine-tuned values for NA, but not provide it for all basins. 

Here are the values I used (empty dics means I used the defaults prior to this pull request):

```
    params_basin = {
        'SA': {},
        'WP': {'max_dspeed_rel' : 0.2,
               'autocorr_ddirection' : 0.75},
        'EP': {'max_dspeed_rel' : 0.2,
               'autocorr_ddirection': 0.255},
        'NA': {'max_dspeed_rel' : 0.2, 
               'autocorr_ddirection' : 0.75, 
               'max_ddirection' : 3.1415/360}, 
        'SP': {'autocorr_ddirection' : 0.5, 
               'max_shift_ini' : 0.3},
        'SI': {},
        'NI': {'max_dspeed_rel'  : 0.2, 
               'autocorr_ddirection' : 0.25,
               'max_ddirection' : 3.1415/90},
        }
```

Reply:
Ok so I've re-run the analysis for the global scale and here are the results:

**Frequency of all bypassing tracks:**
![image](https://user-images.githubusercontent.com/13904718/110634289-1879f000-81aa-11eb-9be1-5f60b393d010.png)

**Frequency of tracks Cat 3-5:**
![image](https://user-images.githubusercontent.com/13904718/110634340-23cd1b80-81aa-11eb-9ccc-e2748bacf0f0.png)

Also, a comparison of the frequency (all tracks) grid cell by grid cell shows that the new parameters greatly reduce the deviation from historical frequencies, shown with standard and log axes (however please note that the grid cells with 0 in historical are not displayed in the log plot, but they are in the other one):
![image](https://user-images.githubusercontent.com/13904718/110638270-9dff9f00-81ae-11eb-8bec-4ff961a00acf.png)
![image](https://user-images.githubusercontent.com/13904718/110638281-a35ce980-81ae-11eb-94c3-bf0375047be8.png)

What do you think? Should change the parameters as I propose, or do you (or somebody else) wants to do a more detailed e.g. basin-specific analysis?

Reply:
Thanks for the global analysis! I think that it is a great improvement to the first default parameters. Hence, please proceed and change the parameters. I am curious to see how they differ from the ones I used.

Reply:
Also please do update the tests accordingly. Thanks!